static/js/base.js

```diff
@@ -148,6 +149,203 @@ function drawObjs(objs, type) {
             ctx.stroke();
         }
     }
+    else if (type === "ramp") {
+        ctx.lineWidth = 2;
+        for (let i = 0; i < objs.length; i++) {
+            if (objs[i].highSide == 90.0) {
```

Comment:
If possible, it would be best to do this use math or by rotating the perspective of the canvas instead of 4 if statements. I would spend some time trying it out, though it may end up not being possible.